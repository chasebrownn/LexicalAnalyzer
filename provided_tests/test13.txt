t1e (((4).(2))*).(((n).(o))|(y)) #
" 42no 4242y no y "