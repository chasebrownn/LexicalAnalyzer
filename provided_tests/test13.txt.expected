t1e , "42no"
t1e , "4242y"
t1e , "no"
t1e , "y"
